src/app/space/analyze/analyze-overview/analyze-overview.component.ts

```diff
@@ -36,8 +45,12 @@ export class AnalyzeOverviewComponent implements OnInit, OnDestroy {
   }
 
   openForgeWizard(addSpace: TemplateRef<any>) {
-    this.selectedFlow = 'selectFlow';
-    this.modalRef = this.modalService.show(addSpace, {class: 'modal-lg'});
+    if (this.authService.isLoggedIn()) {
```

Comment:
@catrobson - this seems to imply that the user sees the add to space button when they aren't authorized to do anything, should we be hiding the button instead?

Reply:
If the user is not logged in, the button shouldn't be shown to them.  I'm not sure what "authorized" means in this context - is that something more than logged in/logged out?

Reply:
The box has been hidden if there is no logged in user, however there is still logic there that will prevent a user from trying to do this action if somehow there is another way to access the code that the button invokes.

Do we want to still leave that code there as a safeguard, or should it be removed?

Reply:
@dgutride Checking in for any input above (sorry, I probably should have highlighted in my post)

Reply:
I'd remove that logic from launching the wizard, I have another PR coming which is doing something similar related to the github linking and provides feedback if they aren't linked.

Reply:
@dgutride To confirm: should the code changes to this file be reset back to being untouched, and then I revisit this later when your PR gets in (and then use what you have added)? Or does your PR take care of everything and I should just remove what I did here and leave it?

Reply:
Your PR will be useful for those not logged in at all.  Hiding the button will still be necessary - my PR will handle what happens if somebody launches the wizard (logged in but missing some permissions)